sdk/src/main/java/com/mapbox/maps/Style.kt

```diff
@@ -381,6 +382,40 @@ class Style internal constructor(
     return styleManager.setStyleLightProperty(id, light)
   }
 
+  /**
+   * Internal use only.
+   *
+   * NO-OP
+   */
```

Comment:
```suggestion
  /**
   * This API is for internal use only and will return a no-op result. 
   *
   */
```

Reply:
Thanks! Much nicer. I've updated the comment.